src/browser/remote_subscriptions.ts

```diff
@@ -49,9 +49,16 @@ interface RemoteSubscriptionProps extends Identity {
 interface RemoteSubscription extends RemoteSubscriptionProps {
     _id: number; // ID of the subscription
     isCleaned: boolean; // helps prevents repetitive un-subscriptions and other cleanup
+    isSystemEvent?: boolean; // helps prevents repetitive un-subscriptions and other cleanup
     unSubscriptions: Map<string, (() => void)[]>; // a map of un-subscriptions assigned to runtime versions
 }
 
+const systemEventsToIgnore: any = {
+    'idle-state-changed': true,
```

Comment:
Should these events go to https://github.com/tgoc99/core/blob/03be9730f3b7ae18be562557808949fd55dd44a7/src/browser/api_protocol/api_handlers/mesh_middleware.ts#L20?

Reply:
In the same object?  Here we refer to system event handlers to ignore and the other file refers to system API methods.  The remote event subscriptions don't go through the same path as the other remote API calls.   They skip the middleware in that file (see ignored 'subscribe to desktop event' in that file) and get handled if necessary in https://github.com/tgoc99/core/blob/03be9730f3b7ae18be562557808949fd55dd44a7/src/browser/api_protocol/api_handlers/event_listener.js. @rdepena correct me if I'm wrong...